firstvoices/backend/serializers/join_request_serializers.py

```diff
@@ -11,32 +12,66 @@
     base_timestamp_fields,
 )
 from backend.serializers.fields import SiteHyperlinkedIdentityField
-from backend.serializers.user_serializers import UserLookupField
+from backend.serializers.user_serializers import UserDetailSerializer
+
+
+class JoinRequestReasonSerializer(serializers.ModelSerializer):
+    reason = fields.EnumField(enum=JoinRequestReasonChoices)
+
+    class Meta:
+        fields = ("reason",)
+        model = JoinRequestReason
 
 
 class JoinRequestDetailSerializer(WritableSiteContentSerializer):
     url = SiteHyperlinkedIdentityField(
         view_name="api:joinrequest-detail", read_only=True
     )
-    user = UserLookupField(required=True, allow_null=False)
+    user = UserDetailSerializer(required=False, allow_null=False, read_only=True)
```

Comment:
Minor but I think if read_only is True you don't need to set required=False here